frontend/src/app/components/transaction/transaction.component.html

```diff
@@ -137,7 +137,6 @@ <h2><a [href]="[ isMempoolSpaceBuild ? '/accelerator' : 'https://mempool.space/a
 
       <app-accelerate-checkout
         *ngIf="(da$ | async) as da;"
-        [cashappEnabled]="cashappEligible"
```

Comment:
`cashappEnabled` default is `true` in `frontend/projects/mempool/src/app/components/accelerate-checkout/accelerate-checkout.component.ts`